lib/blocks.js

```diff
@@ -62,11 +108,12 @@ BlockController.prototype.transformBlock = function(block, info) {
     nonce: blockObj.header.nonce,
     bits: blockObj.header.bits.toString(16),
     difficulty: block.header.getDifficulty(),
-    chainwork: info.chainWork,
-    previousblockhash: blockObj.header.prevHash,
-    nextblockhash: this.node.services.bitcoind.getNextBlockHash(block.hash),
+    chainwork: info.chainwork,
```

Comment:
We are mixing all-lowercase and camelCase here. Should we go ahead and normalize this response object?